state/tx/indexer/kv.go

```diff
@@ -0,0 +1,55 @@
+package indexer
```

Comment:
I'd change the path into state/txindexer/kv/kv.go . 
Also, I'd rename state/tx into state/txindexer .
This would address my other comment about renaming the KV struct into KVIndexer.